compiled_starters/go/.codecrafters/run.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+#
+# This script is used to run your program on CodeCrafters
+# 
+# This runs after .codecrafters/compile.sh
```

Comment:
Some files say "(if required)" on this line, others don't - we need to be consistent